core/src/main/java/org/apache/accumulo/testing/core/randomwalk/image/ImageFixture.java

```diff
@@ -64,15 +63,15 @@ public void setUp(State state, RandWalkEnv env) throws Exception {
     try {
       conn.tableOperations().create(imageTableName);
       conn.tableOperations().addSplits(imageTableName, splits);
-      log.debug("Created table " + imageTableName + " (id:" + Tables.getNameToIdMap(instance).get(imageTableName) + ")");
```

Comment:
I think there are public apis to get the table id

Reply:
Yeah there is, I can use it.